src/bin/zmcontactbackup

```diff
@@ -0,0 +1,565 @@
+                        month          1-12
+                        day of week    0-7 (0 or 7 where 0 is Sunday, or use names like sun, mon, etc...)
+                Example: Everyday at 1am  -  "0 1 * * *"
+        mailbox_server : space separated mailbox server names
+
+=head1 EXAMPLES
+
+Examples :
+1)  # append contact backup schedule to start everyday at 1 am on all mailbox servers
+    $ zmcontactbackup -a "0 1 * * *"
+
+2)  # append contact backup schedule to start everyday at 1 am on "test.server.com" only.
+    $ zmcontactbackup -a "0 1 * * *" test.server.com
+
+3)  # start contact backup on all mailbox servers
+    $ zmcontactbackup -r
+
+4)  # start contact backup on mailbox servers test.server.com and test2.server.com
+    $ zmcontactbackup -r test.server.com test2.server.com
+
+5)  # stop contact backup on all mailbox servers if running.
+    $ zmcontactbackup -p
+
+6)  # stop contact backup on test.server.com, if running. do not disturb running contact backup on other mailbox servers.
+    $ zmcontactbackup -p test.server.com
+
+=head1 DESCRIPTION
+
+This utility is used to schedule contact backup for
+specified mailbox server(s).
+It can be used to start/stop contact backup immediately.
+It uses the ZCS Admin SOAP APIs.
+
+Specifically, this utility was created to meet the following
+requirements:
+1) start contact backup thread
+2) stop contact backup thread
+3) schecule contact backup to run on specific time
+4) contact backup should run on all or specific mailbox servers.
+
+=over 4
+
+=item *
+
+List all existing schedules for contact backup
+This is done by specifying --query option
+
+=item *
+
+Schedule a time to run contact backup on specific or all mailbox servers.
+This is done by specifying --append option with schedule and space separated mailbox server list, if any
+
+=item *
+
+On request start contact backup on specific or all mailbox servers.
+this can be done by specifying --start option with schedule and space separated mailbox server list, if any
+
+=item *
+
+On request stop contact backup on specific or all mailbox servers.
+this can be done by specifying --stop option with schedule and space separated mailbox server list, if any
+
+=back
+
+=head1 Options
+
+The section describes supported options.  The following key is used to
+indicates required arguments, arguments which can be supplied multiple
+times, etc.:
+
+=over 4
+
+=item --help
+
+Display a brief help message.
+
+=item --man
+
+Display full man page.
+
+=item --query
+
+Displays existing contact backup schelude times in cron format with operation and list of mailbox server names.
+
+=item --flush
+
+Flushes all the contact backup schedules.
+
+=item --default
+
+Flushes all the contact backup schedules and sets defalult contact backup schedule.
+Default contact backup shcedule is "0 3 * * *"
+i.e. everyday at 3 am for all the mailbox servers.
+
+=item --append <"schedule"> [<mailbox.server1.name> <mailbox.server2.name> ...]
```

Comment:
since we had other clean, let's go ahead and change ```[<mailbox.server1.name> <mailbox.server2.name> ...]``` here and below to simply be ```[<mailbox.server.name>...]```

Reply:
done